assets/pages/new-contributions-landing/contributionsLanding.scss

```diff
@@ -733,7 +733,7 @@ form {
   color: gu-colour(media-garnett-main-1);
 }
 
-.form__input:focus {
+.form__input:enabled:focus {
```

Comment:
`:enabled` here just makes the selector as [specific](https://specificity.keegan.st/) as the `:enabled:hover` selector on top, meaning focus can take priority